dima/react-movies/src/App.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { Component } from "react";
+import MainPage from "./pages/mainpage";
+import Movies from "./api/movies";
+import "./App.css";
+import MoviePage from "./pages/moviepage";
+import MovieInterface from "./interfaces/movieInterface";
+
+interface AppState {
+  movie: MovieInterface;
+  movies: MovieInterface[];
+  searchedMovies: null | MovieInterface[];
```

Comment:
Can you explain please why did you use additional **searchedMovies** property? It seems like "movies" would be enough. You have some data and then you apply some filtering and sorting on it. NOt really clear why would you need to keep initial data.

Reply:
fixed